app/commands/decidim/kids/create_minor_account.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    # A command to authorize a user with an authorization handler.
+    class CreateMinorAccount < Decidim::Command
+      def initialize(form, current_user)
+        @current_user = current_user
+        @form = form
+      end
+
+      def call
+        return broadcast(:invalid) if form.invalid?
+
+        transaction do
+          create_minor
+          send_email_minor
```

Comment:
We shouldn't send the email here, after creating a minor, we need to redirect to the authorization step